dvc/repo/remove.py

```diff
@@ -2,6 +2,10 @@ def remove(self, target, outs_only=False):
     from dvc.stage import Stage
 
     stage = Stage.load(self, target)
+
+    for o in stage.outs:
+        o.persist = False
```

Comment:
This is a little ugly. How about we pass `force=True` to `stage.remove_outs()` and `stage.remove()` down below?

Reply:
Thats much better.